docs/mp/guides/04_health.adoc

```diff
@@ -529,14 +528,14 @@ kubectl delete -f ./health.yaml
 ----
 
 === Summary
-This guide demonstrated how to use health-check in a Helidon MP application as follows:
+This guide demonstrated how to use health checks in a Helidon MP application as follows:
 
-* Access the default health-check
+* Access the default health check
 * Create and use custom readiness and liveness checks
-* Customize the health-check root path and port
-* Integrate Helidon health-check with Kubernetes
+* Customize the health check root path and port
+* Integrate Helidon health check with Kubernetes
 
-Please refer to the following references for additional information:
+Refer to the following references for additional information:
 
 * MicroProfile health-check specification at https://github.com/eclipse/microprofile-health/releases/tag/2.0
```

Comment:
We've changed the hyphenated "health-check" elsewhere to "health check" and should probably do so on these two lines as well.